tests/framework/jailer.py

```diff
@@ -88,11 +91,12 @@ def construct_param_list(self, config_file, no_api):
             jailer_param_list.extend(['--netns', str(self.netns_file_path())])
         if self.daemonize:
             jailer_param_list.append('--daemonize')
-        if config_file is not None:
-            jailer_param_list.extend(['--'])
-            jailer_param_list.extend(['--config-file', str(config_file)])
-        if no_api:
-            jailer_param_list.append('--no-api')
+        if self.extra_args:
```

Comment:
Could we check here for a `not None` value instead of `false`? This comment also applies for the `value` variable.